src/System.Management.Automation/engine/Modules/ImportModuleCommand.cs

```diff
@@ -1024,7 +1062,10 @@ private PSModuleInfo ImportModule_LocallyViaName(ImportModuleOptions importModul
                 {
                     powerShell.AddCommand("Export-PSSession");
                     powerShell.AddParameter("OutputModule", wildcardEscapedPath);
-                    powerShell.AddParameter("AllowClobber", true);
+                    if (!importModuleOptions.NoClobberExportPSSession)
```

Comment:
Just FYI, the PSCommand API is designed to support the builder pattern so you can save some boilerplate here:

```csharp
    powerShell.AddCommand("Export-PSSession")
        .AddParameter("OutputModule", wildcardEscapedPath)
        .AddParameter("Module", remoteModuleName)
        .AddParameter("FormatTypeName", "*")
        .AddParameter("Force");

    if (!importModuleOptions.NoClobberExportPSSession)
    {
        powerShell.AddParameter("AllowClobber");
    }
```

(Also `AddParameter()` with a single argument adds a switch parameter)

Reply:
good to know.